chatserver.py~

```diff
@@ -3,8 +3,8 @@ import select
 import sys
 
 class Server:
-    def __init__(self):
-        self.host    = ''
+    def __init__(self, ip):
```

Comment:
No need to make changes here. `~py` files are just the swaps. I think `.gitignore` is not working